content/applications/sales/crm/track_leads/lead_distribution_report.rst

```diff
@@ -0,0 +1,262 @@
+from the drop-down menu. This operator **only** includes values greater than (or equal to) the value
+in the third, rightmost field.
+
+The third field on the :guilabel:`Add Custom Filter` pop-up window should contain the earliest date
+leads are selected from.
+
+For example,
+setting `01/01/2024 00:00:00` only includes leads created from, and including, the first day of
+2024.
+
+.. image:: lead_distribution_report/created-on.png
+   :align: center
+   :alt: Add a Created On rule for the start of the year onward.
+
+.. _team-location:
+
+2. Sales team location
+~~~~~~~~~~~~~~~~~~~~~~
+
+Click :guilabel:`New rule` to add a new set of rule fields to the :guilabel:`Add Custom Filter`
+pop-up window. Click the first field for the new rule, and set it to :guilabel:`Sales Team`. Then,
+click the second field of the new rule, and select :guilabel:`contains` from the drop-down menu.
+Selecting this operator filters for any records that contain the words in the third, rightmost
+field.
+
+In this third field, enter the name of the desired sales team(s). For example, setting `us direct sf
+northam` **only** includes sales teams assigned to the U.S., direct, San Francisco, and North
+America.
+
+.. image:: lead_distribution_report/sales-team-location.png
+   :align: center
+   :alt: Use Sales Team to filter the location the lead is associated with.
+
+.. _phone-number:
+
+3. Phone number
+~~~~~~~~~~~~~~~
+
+Click :guilabel:`New rule` to add a new rule to the filter. Set the first field to
+:guilabel:`Phone`. Then, select :guilabel:`is set` from the drop-down menu in the second field.
+Selecting this operator **only** filters for records that have a phone number associated with the
+lead.
+
+.. image:: lead_distribution_report/phone-set.png
+   :align: center
+   :alt: Report only leads with an associated phone number with the Phone field.
+
+.. _active-status:
+
+4. Active status
+~~~~~~~~~~~~~~~~
+
+Click the :guilabel:`Branch` icon to the right of `Phone is set` line, to add a new rule that
+branches from the rules above.
+
+Two horizontal sets of fields appear below a line showing :guilabel:`all of:` with a
+:icon:`fa-caret-down` :guilabel:`(down caret)`. Click the down caret, then select
+:guilabel:`any` from the resulting list.
+
+Set the first field to :guilabel:`Active`. Then, select :guilabel:`is set` in the next field.
+
+.. image:: lead_distribution_report/add-branch.png
+   :align: center
+   :alt: Click the second button to the right of phone is set labeled add branch.
+
+Next, click the :icon:`fa-plus` :guilabel:`New Rule` button next to :guilabel:`Active is set` to
+create a new line of fields beneath it.
+
+Set the first field to :guilabel:`Active`. Then, select :guilabel:`is not set` in the next field.
+This rule will add the activity status of the lead to the report.
+
+.. image:: lead_distribution_report/active-set.png
+   :align: center
+   :alt: Use Active to include active status in the report.
+
+.. _referred-by:
+
+1. Referred by
```

Comment:
the numbers get a little off track here, if I'm not mistaken. worth looking into during your revision